2-AAD-Migration/2-1-SAML-WebApp/README.md

```diff
@@ -0,0 +1,84 @@
+# Migrating a .NET MVC application using SAML protocol from AD FS to Microsoft Active Directory
+
+## Scenario
+
+You have a web application using SAML protocol on AD FS and you would like to migrate it to Microsoft Active Directory.
+
+## About the sample
+
+This sample uses a .NET MVC application configured to use SAML single-sign-on on AD FS, to migrate it to Microsoft Active Directory. The web application used in this sample is the playground project from [chapter 1](https://github.com/Azure-Samples/ms-identity-dotnet-adfs-to-aad/tree/master/1-ADFS-Host/1-1-Setup-SAML-Playground). Ideally, you would like to follow the chapter 1 first to have a testing application for this migration.
+
+After the migration, this sample will use the `App Federation Metadata Url` from the Azure AD application, for the SAML authentication.
+
+### Pre-requisites
+
+- [Visual Studio](https://aka.ms/vsdownload)
+- .NET Framework 4.7.2
+- An AD FS environment
+- An Azure Active Directory (Azure AD) tenant. For more information on how to get an Azure AD tenant, see [How to get an Azure AD tenant](https://azure.microsoft.com/en-us/documentation/articles/active-directory-howto-tenant/)
+
+## Migrate the AD FS SAML application to Azure AD
+
+### Register the SAML application
+
+1. Sign in to the [Azure portal](https://portal.azure.com) using a tenant admin account.
+1. Navigate to the Microsoft identity platform for developers [Enterprise applications](https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/EnterpriseApps) page.
+1. Select **New registration**.
+1. Select **Non gallery app**.
+1. Choose a name for the application, for instance `WebApp_SAML` and select **Add** on the bottom.
+1. Under the Manage section, select **Single sign-on**.
+1. Select **SAML** and the **Set up Single Sign-On with SAML** page will appear.
+
+#### Basic SAML configuration
+
+Learn more about [configuring SAML-based single sign-on on Azure AD](https://docs.microsoft.com/azure/active-directory/manage-apps/configure-single-sign-on-non-gallery-applications).
+
+1. To edit the basic SAML configuration options, select the **Edit** icon (a pencil) in the upper-right corner of the Basic SAML Configuration section.
+1. Set **Identifier (Entity ID)** with an unique URL that follows the pattern, `http://.{your-domain}.com`. For instance: `http://webappsaml.contoso.com`. Copy the Entity ID value to be used in later steps.
```

Comment:
`http://{your-appName}.{your-domain}.com` instead of `http://.{your-domain}.com`?